app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
```

Comment:
- [ ] Hi @Juanse7793 , I'm unable to create a user hence I can't continue with the review. To create a user with rails console returns nil as shown on the screenshot below. Kindly resolve the issue. 

![Screenshot from 2022-07-11 23-50-35](https://user-images.githubusercontent.com/63087491/178372129-0deec6bd-69c7-4ba2-ab89-d52eace658dd.png)